src/extensions/mvs/tree/mvs/mvs-tree-primitives.ts

```diff
@@ -75,6 +75,39 @@ const TubeParams = {
     tooltip: OptionalField(nullable(str), null, 'Tooltip to show when hovering over the tube. If not specified, uses the parent primitives group `tooltip`.'),
 };
 
+const ArrowParams = {
```

Comment:
A suggestion: `ArrowParams` could inherit from `_TubeParams` (in the same way as `DistanceMeasurementParams`). This would mean renaming `tube_radius` -> `radius`, `tube_dash_length` -> `dash_length`.

Or even merge tube and arrow into one primitive type (as an arrow with default params looks the same as a tube), but i understand if you wish to keep them separate.

Reply:
I considered this, but decided to make this its own independent thing instead.